lib/slack-ruby-bot-server/api/presenters/team_presenter.rb

```diff
@@ -10,7 +10,7 @@ module TeamPresenter
         property :team_id, type: String, desc: 'Slack team ID.'
         property :name, type: String, desc: 'Team name.'
         property :domain, type: String, desc: 'Team domain.'
-        property :active, type: Boolean, desc: 'Team is active.'
+        property :active, type: "Boolean", desc: 'Team is active.'
```

Comment:
Does Mongo define `Boolean` but PostgreSQL doesn't. Feels like it should, maybe a monkey patch is good enough for this in the PostgreSQL situation?